src/Framework/FieldsAPI/FormField/FieldStoreAsMeta.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Give\Framework\FieldsAPI\FormField;
+
+/**
+ * @unreleased
+ */
+trait FieldStoreAsMeta {
+
+	/**
+	 * @unreleased
+	 * @var bool
+	 */
+	public $storeAsDonorMeta = false;
```

Comment:
Let's make most properties `protected` as the default. I'd prefer folks to interact with fields declaratively.

Reply:
I went ahead and did this so we can just merge it.